.github/workflows/test-conda.yml

```
@@ -14,23 +14,49 @@ on:
 
 jobs:
   build-linux:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          # - macOS-latest
+          - ubuntu-latest
+        python-version:
+          - "3.8"
+          - "3.9"
+    env:
+      CI_OS: ${{ matrix.os }}
+      PYVER: ${{ matrix.python-version }}
+      PACKAGE: geometric
 
     steps:
       - uses: actions/checkout@v3
 
-      - name: Add conda to system path
+      - name: Set up conda environment
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: devtools/conda-envs/latest.yaml
+          create-args: >- # beware the >- instead of |, we don't split on newlines but on spaces
+            python=${{ matrix.python-version }}
+
+      # Old version commented out 2023-10-25
+      # - name: Add conda to system path
+      #   run: |
+      #     # $CONDA is an environment variable pointing to the root of the miniconda directory
+      #     echo $CONDA/bin >> $GITHUB_PATH
+      # 
+      # # - name: Configure Conda
+      # #   run: conda config --set channel_priority flexible
+      # - name: Install Mamba
+      #   run: conda install -c conda-forge mamba
+      # 
+      # - name: Install Dependencies
+      #   run: mamba env update --file=devtools/conda-envs/latest.yaml --name=base
+
+      - name: Install package
         run: |
-          # $CONDA is an environment variable pointing to the root of the miniconda directory
-          echo $CONDA/bin >> $GITHUB_PATH
-
-      # - name: Configure Conda
-      #   run: conda config --set channel_priority flexible
-      - name: Install Mamba
-        run: conda install -c conda-forge mamba
-        
-      - name: Install Dependencies
-        run: mamba env update --file=devtools/conda-envs/latest.yaml --name=base
+          python -m pip install --no-deps .
+          python -c "import geometric; print(geometric.__version__)"
```

So, now it's saying that it can't import numpy, even though numpy was installed by conda. At first I thought this was a conda env activation problem, but the python and pip commands to install geometric (`  python -m pip install --no-deps .`) finished successfully, and reported paths with `/micromamba/` in them: 

```
  env:
    CI_OS: ubuntu-latest
    PYVER: 3.8
    PACKAGE: geometric
    MAMBA_ROOT_PREFIX: /home/runner/micromamba
    MAMBA_EXE: /home/runner/micromamba-bin/micromamba
    CONDARC: /home/runner/micromamba-bin/.condarc
```

Could you try this to see if numpy is installed+importable at different times?

```suggestion
          python -c "import numpy; print(numpy.__version__)"
          python -m pip install --no-deps .
          python -c "import numpy; print(numpy.__version__)"
          python -c "import geometric; print(geometric.__version__)"
```